server/tests/src/test/java/org/infinispan/server/persistence/CustomStoreIT.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.server.persistence;
+
+import org.infinispan.persistence.spi.NonBlockingStore;
+import org.infinispan.server.test.core.ServerRunMode;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerRuleBuilder;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * @author William Burns &lt;wburns@redhat.com&gt;
+ * @since 11.0
+ **/
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+      CustomStoreOperations.class,
+})
+public class CustomStoreIT {
```

Comment:
Is there a need to create a Suite here? It seems overkill IMO.

Reply:
Fine by me, is there an example of one where it isn't done by a test Suite?

Reply:
Done